src/code_8008C1D0.c

```diff
@@ -303,10 +303,10 @@ void func_8008C73C(Player *player, s8 arg1) {
         D_80165190[2][arg1] = 1;
         D_80165190[3][arg1] = 1;
         D_80165280[arg1] = player->currentSpeed;
-        D_80165480[arg1] = 0;
+        gTimerBoostTripleACombo[arg1] = 0;
         gIsPlayerTripleAButtonCombo[arg1] = FALSE;
-        D_80165440[arg1] = 0;
-        D_80165420[arg1] = 0;
+        gCountAChangement[arg1] = 0;
```

Comment:
What does this mean/do (Changement is not a word)

Reply:
oh my bad x)